params.go

```diff
@@ -181,6 +181,12 @@ func IntToPointer(i int) *int {
 	return &i
 }
 
+// StringToPointer is a function for converting strings into string pointers.
+// This is useful when passing in options to operations.
+func StringToPointer(s string) *string {
```

Comment:
Unless I'm mistaken, one difference is that `StringToPointer` will return `&""`, where `MaybeString` will not.